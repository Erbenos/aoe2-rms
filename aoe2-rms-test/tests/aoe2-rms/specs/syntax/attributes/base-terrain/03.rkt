#lang aoe2-rms
; Create a large clump of forest terrain on grass terrain, then create water on the forest

<TERRAIN-GENERATION>
(create-terrain 'FOREST
                (base-terrain 'GRASS)
                (land-percent 10))

(create-terrain 'WATER
                (base-terrain 'FOREST))
