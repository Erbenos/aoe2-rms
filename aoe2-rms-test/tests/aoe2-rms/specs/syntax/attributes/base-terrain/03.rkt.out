<TERRAIN_GENERATION>
create_terrain FOREST {
  base_terrain GRASS
  land_percent 10
}
create_terrain WATER {
  base_terrain FOREST
}
